tests/wpt/tests/css/CSS2/text/text-align-justify-with-overflow.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+
+<head>
+    <title>CSS Test: Overflowing content with text-align: justify</title>
+    <link rel="author" title="Martin Robinson" href="mrobinson@igalila.com">
+    <link rel="help" href="https://drafts.csswg.org/css-text-3/#text-indent-property">
+    <link rel="help" href="http://www.w3.org/TR/CSS21/text.html#indentation-prop">
+    <link ref="help" href="https://drafts.csswg.org/css-text/#text-align-property">
+    <link rel="match" href="text-align-justify-with-overflow-ref.html"/>
+    <style type="text/css">
+        div {
+            text-align-last: justify;
```

Comment:
This is a CSS2 test, but `text-align-last` is not in CSS2. I would use `text-align` with more text.

Reply:
Fixed.